CHANGELOG.md

```diff
@@ -163,7 +163,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Remove parallel from target options. [#1119](https://github.com/greenbone/gvmd/pull/1119)
 - Remove default port list from CREATE_TARGET [#1151](https://github.com/greenbone/gvmd/pull/1151)
 
-[20.08]: https://github.com/greenbone/gvmd/compare/v9.0.0...gvmd-20.08
+[20.8.0]: https://github.com/greenbone/gvmd/compare/v9.0.0...gvmd-v20.8.0
```

Comment:
```suggestion
[20.8.0]: https://github.com/greenbone/gvmd/compare/v9.0.1...v20.8.0
```

Reply:
i am not sure about the 100 percent correct link for the diff but this one should fit for now.

Reply:
The base should still be v9.0.0 because we branched there but you're right about the "gvmd-" prefix.